consensus/blocknode.go

```diff
@@ -58,143 +41,103 @@ type blockNode struct {
 	delayedSiacoinOutputs map[SiacoinOutputID]SiacoinOutput
 }
 
-// CheckTarget returns true if the block id is lower than the target.
-func (b Block) CheckTarget(target Target) bool {
-	blockHash := b.ID()
-	return bytes.Compare(target[:], blockHash[:]) >= 0
```

Comment:
doesn't seem feasible since the diffing algorithm is pretty dumb.
It'd be cool to have language-specific diffs someday. I'm sure you could make them way more readable if you were actually parsing the syntax.